model/location_credential.rb

```diff
@@ -3,7 +3,7 @@
 require_relative "../model"
 require "aws-sdk-ec2"
 require "aws-sdk-iam"
-require "aws-sdk-sts"
+require "aws-sdk-sts" if Config.test? ? ENV["CLOVER_FREEZE"] != "1" : Config.aws_postgres_iam_access
```

Comment:
I noticed the PR is failing the coverage tests, and it is probably because of this line (I didn't check to confirm). You should be able to fix it by adding `# :nocov:` before and after this line. This fails coverage testing because the second branch is never taken (as `Config.test?` is always true during the tests).